model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -8,6 +8,7 @@
 import functools
 from typing import Any, Callable, Literal, Mapping, Optional, Sequence, TypeAlias, TypeVar
 
+import numpy as np
```

Comment:
Can you add a TODO here that we check whether this works on GPU? You an put my name.
```suggestion
# TODO (@halungge ) test on GPU (NEP 18 ?) 
import numpy as np
```